app/views/decidim/accountability/results/_header.html.erb

```diff
@@ -0,0 +1,6 @@
+
+<p>Te contamos el progreso del Plan de Actuación Municipal ...</p>
```

Comment:
It's important to use translation strings on this kind of messages (for catalan)